src/main/java/government/controller/VehiclesController.java

```diff
@@ -1,36 +1,46 @@
 package government.controller;
 
-
+import government.annotation.Secured;
 import government.dto.TrackerIdDto;
+import government.dto.UserDto;
 import government.dto.VehicleDto;
+import government.facade.OwnershipFacade;
+import government.facade.UserFacade;
 import government.facade.VehicleFacade;
 import government.mapper.TrackerIdMapper;
-import government.model.TrackerId;
-import government.annotation.Secured;
+import government.mapper.VehicleMapper;
 import government.model.Role;
+import government.model.User;
 import government.model.Vehicle;
-import government.mapper.VehicleMapper;
-import jdk.nashorn.internal.objects.annotations.Getter;
 
 import javax.inject.Inject;
+import javax.json.JsonObject;
 import javax.transaction.Transactional;
 import javax.ws.rs.*;
 import javax.ws.rs.core.Response;
-import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
 
 @Path("/vehicles")
 @Produces("application/json")
+@Secured({Role.Employee, Role.Admin})
 public class VehiclesController {
 
     @Inject
     VehicleFacade facade;
 
+    @Inject
+    OwnershipFacade ownershipFacade;
+
+    @Inject
+    UserFacade userFacade;
+
     @Inject
     VehicleMapper vehicleMapper;
```

Comment:
Why is the `VehicleMapper` not part of the `VehicleFacade`? Shouldn't the mapping be done in the facade? Same for `TrackerIdMapper`.